frontend/components/draft/metadata/hooks/useLessonOrder.tsx

```
@@ -2,59 +2,46 @@ import { useRef, useEffect } from "react";
 import { useState, useCallback } from "react";
 import { Droplet } from "@/types";
 import { updateDroplet } from "@/lib/requests/droplet";
```

ESLint

[error] 'updateDroplet' is defined but never used.

(@typescript-eslint/no-unused-vars)